src/ast.hpp

```diff
@@ -216,6 +216,15 @@ struct InsertIntoCommand
   std::vector<std::string> column_names;
   std::vector<LiteralValue> values;
 };
+             
+struct UpdateCommand
+{
+  void execute();
+
+  std::string table_name;
+  std::string column_name;
+  ast::LiteralValue value;
+};
```

Comment:
Just realized you need to add `std::optional<WhereClause> where` in here corresponding to optional WHERE clause.